corehq/apps/mocha/templates/mocha/base.html

```diff
@@ -0,0 +1,34 @@
+{% load hq_shared_tags %}
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
+    <title>CommCareHQ Test</title>
+    <script type="text/javascript" src="{% new_static 'jquery/dist/jquery.min.js' %}"></script>
+    <!-- Mocha and test-only dependencies -->
+    <link href="{% new_static 'mocha/mocha.css' %}" type="text/css" rel="stylesheet"></link>
+    <script src="{% new_static 'mocha/mocha.js' %}"></script>
+    <script src="{% new_static 'chai/chai.js' %}"></script>
+
+    <!-- Core Libraries -->
+    {% block core_libraries %}{% endblock %}
+
+    <!-- App specific dependencies -->
+    {% block dependencies %}{% endblock %}
+
+  </head>
+  <body>
+      <div id="mocha"></div>
+    <script>
+      mocha.setup('bdd')
+      window.assert = chai.assert
```

Comment:
I started writing a response that said

> This is a good point, and a definite pro for BDD style asserts.
>
> I tend to use a convention when I write tests: `assert.equal(actual, expected)` (`actual` always comes before `expected`). But you have to know that convention...

Then I thought about `expect(someOtherValue).toBe(myValue)`. It does not seem obviously wrong (from an English grammar perspective) to say it that way. So I think it's possibly a convention in BDD too, admittedly a tiny bit more obvious which way is the right way.

I think this comes down to a matter of personal preference, which is not worth debating too much more here.